### DISCUSSION TITLE
Will `hurl` support the new HTTP method `QUERY`?

### DISCUSSION THREAD

**Discussion by 0xflotus** [Orange-OpenSource/hurl#1162] [Q&A] [answered]
In the future there will come a new HTTP method: [QUERY](https://www.ietf.org/archive/id/draft-ietf-httpbis-safe-method-w-body-02.html). Will hurl support it soon?

**Comment by jcamiel** [comment, marked as answer]
Hi, yes it makes sense.
As soon as curl supports it and the RFC is adopted, we could add it of course.

**Comment by jcamiel** [comment]
Starting from Hurl 4.0.0 (not released yet), you can use `QUERY` method:

```hurl
QUERY https://foo.com
```

FI, any custom method is also supported.

**Reply by jcamiel** [reply]
4.0.0 is released, with any method supported (`QUERY` for instance)
